server/modules/survey/service/publish/jobs/userPrefsCycleResetJob.js

```diff
@@ -0,0 +1,36 @@
+const R = require('ramda')
+
+const Job = require('../../../../../job/job')
+
+const UserManager = require('../../../../user/manager/userManager')
+const SurveyPublishJobUtils = require('../surveyPublishJobUtils')
+
+class UserPrefsCycleResetJob extends Job {
```

Comment:
what about one job called ```CyclesDeletedCheckJob``` that removes the records and users prefs?

Reply:
good idea